### PR TITLE
fixes expiry policy xml config is shown as null problem at ManCenter

### PR DESCRIPTION
**Pull request by hasancelik**

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/config/ConfigXmlGenerator.java [hunk 1]
```diff
@@ -486,10 +485,23 @@ private void cacheConfigXmlGenerator(StringBuilder xml, Config config) {
...
+        if (!StringUtil.isNullOrEmpty(cacheWriter)) {
+            xml.append("<cache-writer-factory class-name=\"").append(cacheWriter).append("\"/>");
+        }
+    }
+
+    private void checkAndFillCacheLoaderFactoryConfigXml(StringBuilder xml, String cacheLoader) {
+        if (!StringUtil.isNullOrEmpty(cacheLoader)) {
+            xml.append("<cache-loader-factory class-name=\"").append(cacheLoader).append("\"/>");
+        }
+
```
Comment (emrahkocaman) [inline hazelcast/src/main/java/com/hazelcast/config/ConfigXmlGenerator.java]: I think these two methods can be merged into a single one accepting xml tag name as an additional param.

Comment (emrahkocaman) [inline hazelcast/src/main/java/com/hazelcast/config/ConfigXmlGenerator.java]: Nevermind, you can leave it as is.

**Comment by mesutcelik** [comment]
👍

**Comment by mesutcelik** [comment]
can you please send the backport?

**Pull request merged**